add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code lacks a consistent error message display mechanism, which can lead to confusion for users.

Recommendations:
- **Centralize Error Handling**: Create a function or class to handle error messages and display them consistently across the application.
- **Example**:
  ```php
  class ErrorHandler {
      public static function displayError($message) {
          echo "<div class='error'>$message</div>";
      }
  }
  ```
  ```php
  // Usage
  ErrorHandler::displayError("An error occurred: " . $e->getMessage());
  ```